thunder/tests/test_update_aliases.py

```diff
@@ -0,0 +1,217 @@
+from collections.abc import Callable
+
+import pytest
+import torch.testing
+
+import thunder
+import thunder.core.dtypes as dtypes
+import thunder.core.devices as devices
+import thunder.core.prims as prims
+from thunder.core.symbol import Symbol
+from thunder.tests.make_tensor import make_tensor, make_tensor_like
+from thunder.tests.framework import instantiate, ops, NOTHING, TorchExecutor, TorchCompileExecutor, nvFuserExecutor
+from thunder.tests.opinfos import opinfos
+from thunder.tests.test_inplace_functionalization import _inplace_opinfos
+
+
+@ops(_inplace_opinfos, supported_dtypes=(dtypes.float32,))
+def test_update_aliases(op, device, dtype, executor, _):
+    sample = next(op.sample_inputs(device, dtype))
+
+    # polygamma expects an int as its first argument and a tensor as its second but
+    # torch.Tensor.polygamma_ wants opposite; tensor first, int second.
+    # ref:
+    # - https://pytorch.org/docs/stable/special.html#torch.special.polygamma
+    # - https://pytorch.org/docs/stable/generated/torch.Tensor.polygamma_.html
+    args = list(sample.args)
+    if op.name == "polygamma_":
+        args[0], args[1] = args[1], args[0]
```

Comment:
It's a bug in Thunder's `polygamma_`:
https://github.com/Lightning-AI/lightning-thunder/blob/aefafe93f1b859451e3d6120d2775bb699f28372/thunder/torch/__init__.py#L2212
The order of arguments should be the same as in PyTorch.

Reply:
Fix in #1804